src/Analyzer.Core.BuiltInRuleTests/TestTemplates/AppServicesFTPS.badtemplate

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "linux",
+            "name": "linuxKind",
+            "location": "[parameters('location')]",
+            "properties": {
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "api",
+            "name": "withoutSpecifyingProperties",
+            "location": "[parameters('location')]",
+            "properties": {
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "api",
+            "name": "undesiredFtpsState",
+            "location": "[parameters('location')]",
+            "properties": {
+                "siteConfig": {
+                    "ftpsState": "undesiredValue"
+                }
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "api",
+            "name": "ftpsOnly",
+            "location": "[parameters('location')]",
+            "properties": {
+                "siteConfig": {
+                    "ftpsState": "FtpsOnly"
+                }
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "api",
+            "name": "ftpsStateDisabled",
+            "location": "[parameters('location')]",
+            "properties": {
+                "siteConfig": {
+                    "ftpsState": "Disabled"
+                }
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
```

Comment:
you forgot the "config" at the end

Reply:
Should this be flagged in a test?

Reply:
The rule checks for values of ftpsState when specified in a resource of type Microsoft.Web/sites/config, and in one of type Microsoft.Web/sites